### DISCUSSION TITLE
Collapse/Expand Comments with normal click instead of longclock (and make hide toolbar hat the same time)

### DISCUSSION THREAD

**Discussion by Z0pyrus** [Docile-Alligator/Infinity-For-Reddit#1061] [Completed Ideas] [answered]
For myself I would love the feature that you can simply collapse/expend comments with a short click instead of pressing it long or clicking the extra button. I could implement it myself and create a pull request, but I don't know if this feature is welcomed or not, so I'm asking first.
So it could look like this:
![ezgif com-gif-maker](https://user-images.githubusercontent.com/67390601/188271259-2544361c-4bfe-4e0f-9e8a-1384541181b5.gif)

The code that has to be edited is `\infinityforreddit\adapters\CommentsRecyclerViewAdapter.java`, line ~1649

**Comment by Wladefant** [comment, marked as answer]
It is already implemented.

**Reply by Z0pyrus** [reply]
I thought only if you press the comment long? I didn't saw an option for a short click.

**Reply by Wladefant** [reply]
No, also with a short click, i think it is not by default, but can't remember the settings

**Reply by Z0pyrus** [reply]
I don't want to bother you more, but I really can't find it. Only long press works.

**Reply by Wladefant** [reply]
Well, it works for me right now 😁. I will try to find it.

**Reply by Z0pyrus** [reply]
Okay I found it. It was “switch long click and short click” in gesture settings.

**Reply by Wladefant** [reply]
Right
